train.py

```diff
@@ -143,12 +142,13 @@ def get_input_data_tensors(reader,
     files = gfile.Glob(data_pattern)
     if not files:
       raise IOError("Unable to find training files. data_pattern='" +
-                    data_pattern + "'")
-    logging.info("number of training files: " + str(len(files)))
-    filename_queue = tf.train.string_input_producer(files,
-                                                    num_epochs=num_epochs)
+                    data_pattern + "'.")
+    logging.info("Number of training files: %s.", str(len(files)))
+    filename_queue = tf.train.string_input_producer(
+        files, num_epochs=num_epochs, shuffle=True)
     training_data = [
-        reader.prepare_reader(filename_queue) for _ in xrange(num_readers)]
+        reader.prepare_reader(filename_queue) for _ in xrange(num_readers)
```

Comment:
Synced to head. 

Tested with Python3:

python3 youtube-8m-vicaire/train.py --train_data_pattern='/tmp/features/train*.tfrecord' --train_dir=/tmp/features/video_level_logistic_model